charts/hono/values.yaml

```diff
@@ -1457,7 +1457,7 @@ commandRouterService:
 
   # imageName contains the name (excluding registry)
   # of the container image to use for the Command Router service
-  imageName: "eclipse/hono-service-command-router"
+  imageName: "eclipse/hono-service-command-router-infinispan"
```

Comment:
I think the other image references should be adapted as well:
- https://github.com/eclipse/packages/blob/044b6075f84139a00c3e95002ae74e8255598f06/charts/hono/ci/quarkus-native-images-values.yaml#L47
- https://github.com/eclipse/packages/blob/911c8fd57c22c86ecf7f77e67abf5d41e056c219/charts/hono/README.md?plain=1#L610